skgstat/Kriging.py

```diff
@@ -41,7 +46,8 @@ def __init__(
             precision=100,
             solver='inv',
             n_jobs=1,
-            perf=False
+            perf=False,
+            sparse=True
```

Comment:
I would plead for a `False` as the default value here. First, a `True` would change the current behavior. Second, I feel like the use cases in which `sparse=False` makes more sense occur more often. But I might be wrong in that feeling ;) 
We could also keep it `False`for now and turn it to `True` with the next minor version shift, along with a warning that will be raised until then.

Reply:
Hm, that's probably a good idea. Will push a change.